### PR TITLE
D0/D1 sysmes scrutiny

### PR DESCRIPTION
**Pull request by rschlaikjer**
Standardizing titles on no period. Some of these were definitely written before we bumped the size of the textbox.

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -911,39 +911,39 @@ Teach Me, Ciel-Sensei! Ep. 23
...
-Seven years have passed since^then. I wonder how the Tohno^residence has changed. The thing^that's particularly on my mind^is...
-I remember my father, Tohno^Makihisa. He, who hated his^son, is no longer in this world.
-I remember my sister, Akiha. I^wonder how she felt during the^past seven years when I had not^been in contact with her.
-I remember playing with other^children, not just me and Akiha.^I wonder who they were.
-I'm overwhelmed by the grandeur^of the Tohno mansion, which I've^returned to for the first time in^seven years. The one who greeted^me is――
-My memory is hazy and I don't^know what to do. I decide to^follow the girl in silence and^let her guide me around.
-It's rude to call her a 'maid'.^I decide to stop the girl in^front of me and ask for her^name.
-My mind is a mess from the^grandeur of the mansion. I want^to look at the other rooms so I^stop the girl in front of me.
+The end of a familiar routine.^I leave the Arima house, where^I spent the last seven years.^This is the last time I will^see this scenery.
+Maybe there was a time, back^then...
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:917]: JP on this one is 
> そういうコトも、^あったのかもしれないね。

which I assume is a ref to one of the last lines in the file,
> -- 子供の頃、ここを使った事が、あったのかもしれないね
> Maybe there was a time when I used this place as a kid.

and so I've patterned it with that context.

**Review by Petrikow — COMMENTED** [review]
目がくらくらする

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
-I wander around the school.
-I return to the mansion.
+I wander around the school
+I return to the mansion
 Reminiscence
 My father, Tohno Makihisa
-Of course, it's my sister Akiha.
+My sister Akiha, of course
 The other kid who played with us
 Shiki - Homecoming
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:412]: Shiki's Homecoming

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
-I obediently follow her.
-I stop the girl.
-I'll explore the mansion.
-I rest in the living room.
-I'll wait here quietly.
-I'll go for a little walk.
+I obediently follow her
+I stop the girl
+I'll explore the mansion
+I rest in the living room
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:416]: Breather in the living room

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
 Dinner
-I return to my room.
-I have nothing to talk about.
-No choice but to watch carefully.
-Think about it in the corridor.
-Think about it in the corridor.
-I wander around the school.
-Morning Story Continuation
-Afternoon Story Continuation
+I return to my room
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:422]: Returning to my room

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
+I'll figure it out in the hall
+I'll figure it out in the hall
+I wander around the school
+Picking up from this morning
+Picking up from this afternoon
 2nd day ― Morning
-Prof. Arach appears.
-Left the mansion, going to school
+Prof. Arach's Debut
+I leave the mansion for school
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:432]: Going to school from the mansion

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
 The State of the City
 A Change During Homeroom
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:434]: A new kind of homeroom

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
+I'll stay in my room and relax
+Time to turn off the lights
 Agony of Death
 Lunch Break
-I take it easy in the cafeteria.
-Take it easy and buy some bread.
-I go into the hallway and think.
-I go into the hallway and think.
+I'll just eat in the cafeteria
+I'll just buy some bread
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:451]: I'll just eat in the classroom

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
-I go straight home.
-I go to the restaurant.
-A detour before going home
-Why?
-Selinuntius is dead.
-Visit Akiha in the living room.
-I go to Kohaku-san's room.
-I continue to rest in my room.
-Time to turn off the lights.
+I ask after Toyama-sensei
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:436]: about

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
+I decide to drop the subject
+I leave the school
+I'll go straight home
+I'll swing by the restaurant
+I'll take a detour first
+How did I get here...?
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:442]: For whatever reason...

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
+Selinuntius is dead
+I'll go talk to Akiha
+I'll pay Kohaku-san a visit
+I'll stay in my room and relax
+Time to turn off the lights
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:447]: Lights Out

File: system_strings/sysmes_text.en [hunk 11]
```diff
@@ -396,61 +396,61 @@ Teach me, Ciel-sensei!
...
 Agony of Death
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:448]: Death Throes

File: system_strings/sysmes_text.en [hunk 12]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
 Teach Me, Ciel-Sensei! Ep. 24
 Teach Me, Ciel-Sensei! Ep. 25
 
-The next thing I knew, I found^myself in a hospital bed. The^world I see is all covered in^lines.
+The next thing I know, I find^myself in a hospital bed. The^world around me is covered^in lines.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:914]: When I come to, I find myself in a hospital bed. The world around me is scrawled with lines.
(Change scene too)

File: system_strings/sysmes_text.en [hunk 13]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
-I find myself in a back alley^in the city. The curfew has long^passed. I have to get back to the^mansion as soon as possible.
-I'm unable to return home before^sunset. The tyrant summons me to^her office, and Melos is furious.
-I pissed off Akiha today. I head^to the living room to see if we^could talk, but the person I run^into there is...
-It's still too early to go to^bed. Should I visit Kohaku-san's^room and ask her if I could use^her TV...?
-I pass the time in my room. While^I was relaxing, I hear a voice^call me through the door.
-It's ten o'clock, time for lights^out. I don't have anything else^to do. I'll just lie down^quietly.
+Dinner with the family head has^begun! The air is electric. I^need to learn table manners^fast, or else!
+I survive my ordeal and make it^back to my room. Something's^been bothering me, so I ask^Hisui about it once she^finishes making the bed.
+I run into Ciel-senpai, a^school celebrity. Sadly, I have^nothing to talk to her about.
+I'm distracted by her legs,^which are as toned as those of^a track and field athlete. I^wonder if she trains every day?
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:941]: drop track and field

File: system_strings/sysmes_text.en [hunk 14]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
-My mind is a mess from the^grandeur of the mansion. I want^to look at the other rooms so I^stop the girl in front of me.
+The end of a familiar routine.^I leave the Arima house, where^I spent the last seven years.^This is the last time I will^see this scenery.
+Maybe there was a time, back^then...
+I actually do have a question^for her. This is as good an^opportunity as any, so I may as^well ask it.
+Ciel-senpai's in a rush, so I^see her off at the station.^After that, I head to school at^my own pace.
+I'll have lunch with Arihiko in^the cafeteria today. Apparently^he's invited a special guest.
+My mind's still a mess. As I'm^sitting around trying to pull^myself together, Yumizuka^Satsuki calls out to me.
+My thoughts remain clouded, but^I can't stay in this classroom^forever. I wonder what I should^do now.
+I'll kill some more time. As I^stroll around the school^grounds, I remember an incident^from my middle school days.
+I make up my mind and leave the^school. It's been a long time,^but the route still feels^familiar. As I walk, I think^back on my memories of the^Tohno mansion.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:924]: I work up my courage...

File: system_strings/sysmes_text.en [hunk 15]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+It's been seven years... I^wonder how the Tohno mansion^has changed. If there's one^thing in particular I remember,^it's―――
+I remember my father, Tohno^Makihisa. He was a strict man,^and disapproved of his son. And^now, he is no longer with us.
+I remember my sister, Akiha. I^haven't talked to her at all^since I left... I wonder what's^been going through her head^these past seven years.
+I definitely remember Akiha and^I playing with another child^back then... I wonder who they^were.
+I return to the Tohno mansion^for the first time in seven^years. It's grandeur is^honestly overwhelming. I'm^greeted by―――
+My memories are still hazy, and^I don't want to accidentally^put a foot wrong. I'll just^keep my mouth shut and let her^show me around.
+It would be rude to just refer^to her as 'maid'. I stop the^girl in front of me and ask for^her name.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:931]: 'you there'.

File: system_strings/sysmes_text.en [hunk 16]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+Dinner with the family head has^begun! The air is electric. I^need to learn table manners^fast, or else!
+I survive my ordeal and make it^back to my room. Something's^been bothering me, so I ask^Hisui about it once she^finishes making the bed.
+I run into Ciel-senpai, a^school celebrity. Sadly, I have^nothing to talk to her about.
+I'm distracted by her legs,^which are as toned as those of^a track and field athlete. I^wonder if she trains every day?
+Should I eat in the cafeteria,^or pick something up at the^store? As I'm trying to make up^my mind, I see a familiar^face...
+I'd hoped for a serendipidous^encounter, but no such luck. I^guess I'll go to the cafeteria^after all.
+I remember the promise from^this morning. I'll stop by the^tea ceremony clubroom. I'm sure^Ciel-senpai will be there.
+Apparently Senpai wants to get^to know me better. I don't know^if I can live up to her^expectations, but I'll try^talking about recent events.
+Apparently Senpai wants to get^to know me better. I feel like^it's a boring subject, but I'll^talk a bit about myself.
+I wake up from a deep sleep.^This marks the beginning of a^new day at the Tohno mansion.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:947]: from my slumber.

File: system_strings/sysmes_text.en [hunk 17]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+Akiha and I don't have much to^talk about, so we pass the time^in comfortable silence.^Until―――
+It's my first day commuting to^school from the Tohno mansion.^I feel like a transfer student^as I walk down the street. The^city looks the same as always,^but...
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:949]: almost feel

File: system_strings/sysmes_text.en [hunk 18]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+It doesn't look like Arihiko's^shown up to class today. As I^scroll through my phone, I let^my mind wander.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:950]: Drop doesn't look like

File: system_strings/sysmes_text.en [hunk 19]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+It's still too early to go to^bed. I wonder if Kohaku-san^would let me use the TV in her^room...?
+I pass the time in my room.^While I'm relaxing, someone^knocks on my door.
+It's ten o'clock, time for^lights out. I don't really have^anything else to do, so I guess^I'll go to sleep.
 I hear the sound of howling from^somewhere. It's so annoying that^I can't sleep...
-It's time for lunch. Despite^the introduction of Noel-sensei,^things remain the same.
-I feel somewhat good today. I'll^splurge and order the 'A' set^meal at the cafeteria. I'm rich.
-Perhaps I should refrain from^wasting money. When I come back^from buying bread at the store,^Yumizuka calls out to me.
-I run into Ciel-senpai in the^hallway. She said she bought too^much lunch. I'll take her up on^her offer.
-I encounter something strange^on the stairs. The new teacher,^Noel-sensei, seems to be at a^loss by the wall.
+It's time for lunch.^Noel-sensei may have caused a^stir, but some things remain^the same.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:966]: some things never change

File: system_strings/sysmes_text.en [hunk 20]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+I feel like I'm in good shape^today. I'll treat myself to a^lavish 'Meal Set A' at the^cafeteria. I've got money to^burn.
+I'd better keep an eye on my^expenses... When I come back^from buying bread at the school^store, I'm approached by^Yumizuka.
+I run into Ciel-senpai in the^hallway. She says she's bought^too much food for lunch. I take^her up on her kind offer.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:969]: she says -> apparently

File: system_strings/sysmes_text.en [hunk 21]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+School is over for the day. As^I leave the classroom, I bump^into the woman everyone's^talking about: Noel-sensei.
+I wonder what she means by 'end^up like Toyama-sensei'... I^decide to stop her and ask.
+I wonder what she means by 'end^up like Toyama-sensei'... but^I decide not to ask.
+I say goodbye to Noel-sensei^and head out of the school^gates. I could just go straight^back to the mansion, but maybe^I should stop somewhere else^along the way?
+The head of the family told me^I had to be back before sunset.^I should probably just do as^she says.
+I just remembered that^Yumizuka-san gave me a point^card. It would be a shame for^it to go to waste, maybe I'll^go pay her a visit.
+I'll kill some time before I^head back. I wonder if I should^hit up the shops around the^station, or go to Arihiko's^house...
+When I come to my senses, I'm^in a back alley downtown.^Curfew has long since passed. I^need to get back to the mansion^as soon as possible.
+I failed to make it home before^sunset. The tyrant summons me^to her office, and Melos is^furious.
+I've already messed up and^angered Akiha. I go to the^living room in hope of talking^to her, but instead―――
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:961]: but the person who greets me there is..

File: system_strings/sysmes_text.en [hunk 22]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+It's still too early to go to^bed. I wonder if Kohaku-san^would let me use the TV in her^room...?
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:962]: watch TV in her room?

File: system_strings/sysmes_text.en [hunk 23]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
+I pass the time in my room.^While I'm relaxing, someone^knocks on my door.
+It's ten o'clock, time for^lights out. I don't really have^anything else to do, so I guess^I'll go to sleep.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:964]: At ten o'clock, the mansion's lights turn off.

File: system_strings/sysmes_text.en [hunk 24]
```diff
@@ -911,63 +911,63 @@ Teach Me, Ciel-Sensei! Ep. 23
...
 I hear the sound of howling from^somewhere. It's so annoying that^I can't sleep...
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:965]: It worms its way into my ears, disturbing my sleep.

**Pull request merged**